components/push/src/communications.rs

```diff
@@ -418,43 +417,43 @@ mod test {
     const SENDER_ID: &str = "FakeSenderID";
     const SECRET: &str = "SuP3rS1kRet";
 
-    /*
-        # While this is technically dead code, it should not be removed.
-        # This test case can be used against a locally running autopush server
-        # to verify that communications functions are operating correctly.
-        # Since it's not really feasible to run a local autopush server on
-        # a CI machine, it's reserved only for local debugging and testing.
-        #[test]
-        fn test_live_server() {
-            let config = PushConfiguration {
-                http_protocol: Some("http".to_owned()),
-                server_host: "localhost:8082".to_owned(),
-                sender_id: "testing".to_owned(),
-                bridge_type: Some("test".to_owned()),
-                registration_id: Some("SomeRegistrationValue".to_owned()),
-                ..Default::default()
-            };
-
-            let mut conn = connect(config, None, None).unwrap();
-            let sub1 = conn.subscribe(DUMMY_CHID).unwrap();
-            println!("### conn: {:?}", (&conn.uaid, &conn.auth));
-            println!("### Sub1: {:?}", sub1);
-            let sub2 = conn.subscribe(DUMMY_UAID).unwrap();
-            println!("### Sub2: {:?}", sub2);
-            println!("### conn: {:?}", (&conn.uaid, &conn.auth));
-            let ll = conn.channel_list().expect("channel list failed");
-            println!("### channels: {:?}", ll);
-            conn.unsubscribe(Some(DUMMY_UAID))
-                .expect("chid unsub failed");
-            println!(
-                "### verify: {:?}",
-                conn.verify_connection(&[DUMMY_CHID.to_owned()])
-            );
-            println!("### channels: {:?}", conn.channel_list());
-            conn.unsubscribe(None).expect("uaid unsub failed");
+    /* LIVETEST
```

Comment:
Shouldn't we just put this code in a cargo example instead?

Reply:
Ah! I didn't know about `cargo run --example`.